packages/shared/src/telemetry.ts

```diff
@@ -186,18 +204,18 @@ export class TelemetryCollector {
         'Content-Type': 'application/json',
       },
     })
-      .catch(err => console.error(err))
+      .catch(() => void 0)
```

Comment:
Swallowing errors here to ensure we don't create noise as a result of our telemetry.